beta/src/components/DocsFooter.tsx

```diff
@@ -66,9 +69,15 @@ function FooterLink({
   title: string;
   type: 'Previous' | 'Next';
 }) {
+  let target = '';
+  if (href.startsWith('https://')) {
+    target = '_blank';
+  }
+
   return (
     <NextLink href={href}>
       <a
+        target={target}
```

Comment:
Shouldn't target be added only when we have href to be url? if else we wont need target attribute at all. Lets not have target to be empty string.
<img width="70" alt="Screenshot 2022-10-13 at 3 31 14 PM" src="https://user-images.githubusercontent.com/32865581/195567253-723f0515-301c-44aa-83ff-8f2c15f3c08f.png">

Reply:
I have corrected the target problem in the FooterLink component and also in the SidebarLink component that also had the same error. 

![image](https://user-images.githubusercontent.com/105958505/195612130-86c21010-ca20-49d9-9283-c8afd80ee099.png)